R/chartutils.R

```diff
@@ -102,17 +103,24 @@ checkMatrixNames <- function(x, assign.col.names = TRUE)
         dimnames(new.x) <- old.names
     }
 
-    # Many of the charts use the statistic attribute
-    x <- CopyAttributes(new.x, x)
+    # Convert percentage data to decimal form
+    stat <- attr(x, "statistic")
+    if (is.null(stat) && !is.null(dimnames(x)))
+        stat <- dimnames(x)[[length(dim(x))]][1]
```

Comment:
Does the final dimension always contain the statistics?

Reply:
Yep, if the statistic attribute is NULL, then its always in the final dimension